src/main/java/edu/cornell/kfs/vnd/CUVendorPropertyConstants.java

```diff
@@ -40,4 +40,6 @@ public class CUVendorPropertyConstants {
     
     public static final String VENDOR_HEADER_PREFIX = "vendorHeader.";
     public static final String VENDOR_ADDRESS_STATE_CODE = "vendorAddresses.vendorStateCode";
+    
+    public static final String VENDOR_DETAIL_ACTIVE_INDICATOR = "activeIndicator";
```

Comment:
this constants exists in CuVendorDaoOjb ... should that constant be used so we don't have duplication?

Reply:
I agree with your idea.  Maybe  it would make sense to fix it in the same user story we look at CUVendorService